README.md

```diff
@@ -99,138 +99,155 @@ outputs, and adhere to any given constraints.
+   ```
 
 3. Install GraphViz
-```bash
-brew install graphviz
-```
-
-4. Install python packages
-```bash
-pip install -r requirements.txt
-```
-
-5. Set up Discord bot
-```bash
-export DISCORD_TOKEN={token from discord developer portal}
-```
-
-6. Start the bot
-```bash
-python bot.py
-```
+   ```zsh
+   brew install graphviz
+   ```
+
+1. Clone repo (in directory of you choosing)
+
+   ```zsh
+   git clone https://github.com/ScottJDaley/ada.git
+   ```
+
+   :bulb: MacOS might prompt you to download `Xcode Commandline Tools` the first time you use the `git` command
+
+2. Move inside the repo
+   ```zsh
+   cd ada
+   ```
+
+1. Create python virtual environment and activate it
+   ```zsh
+   python3 -m venv venv && source ./venv/bin/activate
+   ```
+
+   :bulb: You may deactivate the virtual environment with `deactivate`
+
+2. Install python requirements
+   ```zsh
+    pip install -r requirements.txt
+   ```
+
+
+
+4. Set up Discord bot
+   ```bash
+   export DISCORD_TOKEN={token from discord developer portal}
+   ```
+
+5. Start the bot
+   ```bash
+   python bot.py
+   ```
  
 ### Linux Installation
 
 1. Install python3
-```console
-sudo apt-get install python3
-```
+   ```bash
+   sudo apt-get install python3
+   ```
 
 2. Install pip
-```console
-sudo apt-get install python3-pip
-```
+   ```bash
+   sudo apt-get install python3-pip
+   ```
 
 3. Install GraphViz
-```console
-sudo apt-get install graphviz
-```
+   ```bash
+   sudo apt-get install graphviz
+   ```
 
 4. Install git
-```console
-sudo apt-get install git
-```
+   ```bash
+   sudo apt-get install git
+   ```
 
 5. Clone repo (in directory of you choosing)
-```console
-git clone https://github.com/ScottJDaley/ada.git
-```
+   ```bash
+   git clone https://github.com/ScottJDaley/ada.git
+   ```
 
 6. Move inside the repo
-```console
-cd ada
-```
+   ```bash
+   cd ada
+   ```
+
+7. Create Virtual environment and activate it
+   ```bash
+   python3 -m venv venv && source venv/bin/acivate
+   ```
 
-7. Install python packages
-```console
-pip3 install -r requirements.txt
-```
+   :bulb: You may deactivate the virtual environment with `deactivate`
+
+7. Install python packages 
+   ```bash
+   pip3 install -r requirements.txt
+   ```
 
 8. Set up Discord bot
-```console
-echo "DISCORD_TOKEN={token from discord developer portal}" > .env
-```
+   ```bash
+   export DISCORD_TOKEN={token from discord developer portal}
+   ```
 
 9. Start the bot
-```
-python3 bot.py
-```
- 
-### How to run
+   ```bash
+   python3 bot.py
+   ```
 
-#### Command line tool
 
-1. Double click `tool.py` to run it.
+#### Usage
+1. Ensure the bot is running and was started with a valid `DISCORD_TOKEN` as described above.
+2. Invite the bot to your server using a link generated from OAuth2 page on https://discord.com/developers/applications.
+3. The bot should be able to respond to commands in Discord after a few seconds.
+4. Press enter in the terminal to shut down the bot.
+
+#### Command line tool
```

Comment:
I might have goofed up here. I don't actually know what `tool.py` is or if it is still valid. Is it just a convenience script for running the bot a different way?